src/commandline_options.cpp

```diff
@@ -239,6 +239,7 @@ commandline_options::commandline_options(const std::vector<std::string>& args)
 		("log-precise", "shows the timestamps in log output with more precision.")
 		("no-log-to-file", "log output is written only to standard error rather than to a file. The environment variable WESNOTH_NO_LOG_FILE can also be set as an alternative.")
 		("log-to-file", "log output is written to a file. Cancels the effect of --no-log-to-file whether implicit or explicit.")
+		("no-log-sanitize", "do not sanitize usernames in log")
```

Comment:
Just repeating the words doesn't make good documentation, it's better to try to avoid using the same words.

"disable the anonymization that's normally applied to log files, for example replacing usernames with “USER”"

Reply:
"log files" doesn't quite fit in my opinion, since it's also applied when logging to terminal. I'd just say "logging" or "logs".

Reply:
Used stevecotton's suggested message with 'to log files' replaced by 'when logging'.

Reply:
Also made previous option (`log-to-file`) a bit more verbose.